src/generic/grideditors.cpp

```diff
@@ -1266,14 +1245,26 @@ void wxGridCellBoolEditor::SetSize(const wxRect& r)
 
     // position it in the centre of the rectangle (TODO: support alignment?)
 
+    wxRect cellRect = r;
```

Comment:
I'd rather keep the old name to avoid the changes in the diff below related to just renaming. If you'd like, let's rename it as a separate commit later, but for now this just makes things less clear.